instrumentation/aws_sdk/opentelemetry-instrumentation-aws_sdk.gemspec

```diff
@@ -27,6 +27,7 @@ Gem::Specification.new do |spec|
 
   spec.add_dependency 'opentelemetry-api', '~> 1.0'
   spec.add_dependency 'opentelemetry-instrumentation-base', '~> 0.19.0'
+  spec.add_dependency 'opentelemetry-semantic_conventions'
```

Comment:
should there be a version range for that dependency?
[like here](https://github.com/open-telemetry/opentelemetry-ruby/blob/dab6dda9fb1a7fdc2df571f0ad4e2a35cb39b4e0/instrumentation/trilogy/opentelemetry-instrumentation-trilogy.gemspec#L30) for example

Reply:
If there is a min version of semconv that works for awssdk, then yes we should have a min version, but i wasn't aware of what that was.